promise/tests/test-promise.js

```diff
@@ -131,7 +131,7 @@ o.spec("promise", function() {
 				callAsync(function() {resolve(promise)})
 			})
 
-			promise.then(null, done)
+			promise.then(null, () => done())
```

Comment:
My one and only nit: use ES5-style functions.

Reply:
I converted one arrow fn in ospec to a fn-expression. Left the tests untouched.